src/agent/communicator/include/ihttp_socket.hpp

```diff
@@ -23,6 +23,9 @@ namespace http_client
 
         virtual void Read(boost::beast::http::response<boost::beast::http::dynamic_body>& res) = 0;
 
+        virtual void ReadToFile(boost::beast::http::response_parser<boost::beast::http::dynamic_body>& res,
```

Comment:
Wouldn't it be simpler to return a string? Instead of passing a reference. I'm also confused about the type and name for the parameter here, if it is an output file, shouldn't it be mutable? Why is it const?

Reply:
I changed the name of the parameter, this indicates the path and name of the file where it will be saved.